README.md

```diff
@@ -39,6 +39,8 @@ bench
   });
 
 await bench.run();
+
+// view, print, process bench.results
```

Comment:
let's remove this comment here, the note is enough.

Reply:
That's also good!

Reply:
try getting the task (with bench.getTask) and log its mean or hz! that example here is good

Reply:
btw, it'd be good if we can convert the require call in the example to an import call! because it's incorrect! Thanks.

Reply:
done